app/src/main/java/com/arctouch/codechallenge/api/TmdbApi.kt

```diff
@@ -10,31 +11,32 @@ import retrofit2.http.Query
 
 interface TmdbApi {
 
-    companion object {
-        const val URL = "https://api.themoviedb.org/3/"
-        const val API_KEY = "1f54bd990f1cdfb230adb312546d765d"
-        const val DEFAULT_LANGUAGE = "pt-BR"
-        const val DEFAULT_REGION = "BR"
-    }
-
     @GET("genre/movie/list")
     fun genres(
-        @Query("api_key") apiKey: String,
-        @Query("language") language: String
+            @Query("api_key") apiKey: String,
```

Comment:
Here you are repeating to pass `api_query` and `language` on `genres()`, `upcomingMovies()`, and also on `topRatedMovies()`. You should had void them by using HttpInterceptors, as you did for logging. In that way, each request should be intercepted and add those parameters before it is sent. For further reading on that, please take an eye on https://github.com/square/okhttp/wiki/Interceptors.

Reply:
yes, you're right again, thank you for your tips.